modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BoundaryEventJsonConverter.java

```diff
@@ -87,9 +89,24 @@ protected void convertElementToJson(ObjectNode propertiesNode, BaseElement baseE
         ObjectNode dockNode = objectMapper.createObjectNode();
         GraphicInfo graphicInfo = model.getGraphicInfo(boundaryEvent.getId());
         GraphicInfo parentGraphicInfo = model.getGraphicInfo(boundaryEvent.getAttachedToRef().getId());
-        dockNode.put(EDITOR_BOUNDS_X, graphicInfo.getX() - parentGraphicInfo.getX());
-        dockNode.put(EDITOR_BOUNDS_Y, graphicInfo.getY() - parentGraphicInfo.getY());
+        BigDecimal parentX = new BigDecimal(parentGraphicInfo.getX());
```

Comment:
the same logic should be used in org.flowable.cmmn.editor.json.converter.CriterionJsonConverter#convertElementToJson
Are you going to add it there?